certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/documents/{document-id}/status/aborted": {
+      "post": {
+        "operationId": "PostEnvelopesEnvelopeIdDocumentsDocumentIdStatusAborted",
+        "summary": "Set document abort status",
+        "description": "Sets the status of a given document to aborted",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/DocumentId"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelope-descriptors/{envelope-descriptor-id}": {
+      "get": {
+        "operationId": "GetEnvelopeDescriptorsEnvelopeDescriptorId",
+        "summary": "Get descriptor by id",
+        "description": "Gets information of a given descriptor by its Id",
+        "tags": [
+          "cleaned"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeDescriptorId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Descriptor"
+            }
+          },
+          "404": {
+            "$ref": "#/responses/404"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/trash-status": {
+      "put": {
+        "operationId": "PutEnvelopesEnvelopeIdTrashStatus",
+        "summary": "Set envelope trash status",
+        "description": "Sets the status of a given envelope to trashed",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "comment": {
+                  "type": "string",
+                  "title": "Comment"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/envelope-trash